tests/foreman/virtwho/test_ui_virtwho.py

```diff
@@ -689,3 +693,36 @@ def test_positive_overview_label_name(form_data, session):
             assert results['overview'][key] == value
         session.virtwho_configure.delete(name)
         assert not session.virtwho_configure.search(name)
+
+
+def test_positive_last_checkin_status(form_data, session):
+    """Verify the Last Checkin status on Content Hosts Page.
+
+    :id: 3931e10e-8adc-4ca4-8963-20572b2f99bf
+
+    :expectedresults: The Last Checkin time on Content Hosts Page is UTC time
+
+    :BZ: 1652323
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Medium
+    """
+    name = gen_string('alpha')
+    form_data['name'] = name
+    with session:
+        session.virtwho_configure.create(form_data)
+        values = session.virtwho_configure.read(name)
+        command = values['deploy']['command']
+        hypervisor_name, guest_name = deploy_configure_by_command(command, debug=True)
+        time_now = datetime.utcnow()
+        assert session.virtwho_configure.search(name)[0]['Status'] == 'ok'
```

Comment:
Why search for configuration and read status from search results, when status is already in `values`?

If you decide to apply my previous comment, here's how to read two widgets at once:
```python
values = session.virtwho_configure.read(name, widget_names=['deploy.command', 'overview.status'])
command = values['deploy']['command']
status = values['overview']['status']
```

Reply:
No really, the status of virt-who will update after we deploy it, so we should read it again here